### ISSUE TITLE
Feature: load playlists from network folder

### ISSUE THREAD

**Issue by moretocome** [MusicPlayerDaemon/MPD#593]
## Feature request
I store my music on a network folder, and I would like to store my playlists in the same directory. But the ``````smbclient`````` and ``````nfs`````` storage plugins are only available for use with the ``````music_directory`````` setting.

Would it be possible to also enable those plugins for the ``````playlist_directory`````` setting? I know I would be able to mount the playlist folder on boot through ``````fstab``````. But it seems simpler if MPD is able to use ``````smbclient`````` directly.

Thanks in advance and for at great piece of software in any case!

**Comment by eMPee584** [comment]
[…]
You aware that any playlists will work fine from within the Music folder? :smile:

**Comment by moretocome** [comment]
Yes of course, @eMPee584. The Problem is storing them there. It has to be done manually even though MPD has the ability to create playlists automatically :)
